packages/react-native/src/utils/pod-install-task.ts

```diff
@@ -70,15 +74,15 @@ export function podInstall(
         stdio: 'inherit',
       });
     }
-    execSync(
-      `pod install ${options.repoUpdate ? '--repo-update' : ''} ${
-        options.deployment ? '--deployment' : ''
-      }`,
-      {
-        cwd: iosDirectory,
-        stdio: 'inherit',
-      }
-    );
+    const podCommand = [
+      options.useBundler ? 'bundle exec' : 'pod install',
```

Comment:
question: should the command be `bundle exec pod install`? from the logic, if the useBundler is true, it only contains `bundle exec`.
should it be:
```
options.useBundler ? 'bundle exec' : '',
'pod install',
```

Reply:
Yeah, my mistake. Fixing it